ui_framework/src/components/expression/expression_item_popover.test.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { render } from 'enzyme';
+import { requiredProps } from '../../test/required_props';
+
+import {
+  KuiExpressionItemPopover,
+  POPOVER_ALIGN
+} from './expression_item_popover';
+
+describe('KuiExpressionItemPopover', () => {
+  test('renders', () => {
+    const component = (
+      <KuiExpressionItemPopover
+        title={'title'}
```

Comment:
This works, but the canonical form would be `title="title"`. The same applies to some of the other props below.

Reply:
@weltenwort OK. I hope I changed that in all files.